# List of published bioinformatics protocols for wastewater NGS data analysis

1. [lineagespot](https://github.com/BiodataAnalysisGroup/lineagespot), [bioconductor package](https://bioconductor.org/packages/3.15/bioc/html/lineagespot.html), and [article](https://www.nature.com/articles/s41598-022-06625-6)
2. [PiGx SARS-CoV2 wastewater sequencing pipeline](https://github.com/BIMSBbioinfo/pigx_sarscov2_ww) (BIMSBbioinfo)
3. [COWWID](https://github.com/cbg-ethz/cowwid), [Detection of SARS-CoV-2 variants in Switzerland by genomic analysis of wastewater samples](https://www.medrxiv.org/content/10.1101/2021.01.08.21249379v2)
5. [Wastewater based variant tracking for SARS CoV 2](https://www.cdc.gov/amd/training/covid-toolkit/module2-7.html) CDC Module 2.7
6. [Rapid and inexpensive whole-genome sequencing of SARS-CoV-2 using 1200â€‰bp tiled amplicons and Oxford Nanopore Rapid Barcoding](https://academic.oup.com/biomethods/article/5/1/bpaa014/5873518), Biology Methods and Protocol
7. [Artic Network: nCoV-2019 novel coronavirus bioinformatics protocol](https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html)
8. [Looking for a needle in a haystack. SARS-CoV-2 variant characterization in sewage](https://www.sciencedirect.com/science/article/pii/S2468584421000805?via%3Dihub)
9. [The Rapid Assessment of Aggregated Wastewater Samples for Genomic Surveillance of SARS-CoV-2 on a City-Wide Scale](https://www.mdpi.com/2076-0817/10/10/1271)
10. [Public Health Alliance for Genomic Epidemiology](https://github.com/pha4ge). There is a corresponding mapping to the ENA (https://github.com/pha4ge/SARS-CoV-2-Contextual-Data-Specification)
12. Kallisto-Approach, [Lineage abundance estimation for SARS-CoV-2 in wastewater using transcriptome quantification techniques](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02805-9), [GitHub](https://github.com/baymlab/wastewater_analysis)
